modules/compute/vm-instance/variables.tf

```diff
@@ -135,6 +135,12 @@ variable "service_account" {
   }
 }
 
+variable "set_bucket_viewer" {
```

Comment:
I am feeling that this variable needs to indicate some reference to startup_script. It is likely not clear that vm-instance, or batch-job-template, or others are creating a bucket. What do you think about something like `grant_startup_script_bucket_viewer`. I guess as I am brainstorming here, `set` also seems like an odd term in the name. I take your meaning to be "set this property on the sub module", but users are probably not thinking about the submodule. The actually effect is granting a permission or binding a role or what have you.

Reply:
I share a bit of the concern and did not have great answer myself. I chose to align `set` with terminology in the CFT module:

https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/master/variables.tf#L155

It also had advantage of landing near `service_account_email` in the README. WDYT of `set_startup_script_bucket_viewer`?

I am anticipating that, eventually, most usage of startup-scripts module will be behind the scenes (as it is here) so I agree this is an important issue.

Reply:
Resolving due to lower scope of PR